addons/binding/org.openhab.binding.silvercrestwifisocket/src/main/java/org/openhab/binding/silvercrestwifisocket/internal/enums/SilvercrestWifiSocketVendor.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.silvercrestwifisocket.internal.enums;
+
+/**
+ * This enum represents the available Wifi Socket vendors.
+ *
+ * @author Christian Heimerl - Initial contribution
+ *
+ */
+public enum SilvercrestWifiSocketVendor {
+    Lidl_Silvercrest("C1", "7150"),
+    Aldi_EasyHome("C2", "92DD");
+
+    private String companyCode;

```

Comment:
Prefer `final` for fields where possible.

Reply:
Will do it...